server/app.js

```diff
@@ -36,4 +42,6 @@ app.use(function(err, req, res, next) {
   res.json({ error: err });
 });
 
+app.listen(3001)
+
```

Comment:
this shouldn't be necessary. You can run the server with `npm run dev` without any changes to the server configuration